tests/test_element.py

```diff
@@ -77,16 +77,25 @@ def test_electrophilicity(symbol):
     e.electrophilicity()
 
 
-def test__eq__():
+def test_eq():
 
     elements = get_all_elements()
     for e in elements:
         clone = element(e.symbol)
         assert clone == e
 
 
-def test__ne__():
+def test_ne():
 
     elements = get_all_elements()
     for e1, e2 in zip(elements[:-1], elements[1:]):
         assert e1 != e2
+
+    assert element(12) != "sample string"
+
+
+def test_hashable():
```

Comment:
👍  to having this tested, whether or not the changes to the source code are accepted